src/frontend/js/login.js

```diff
@@ -149,11 +217,38 @@ div:has(#submit_btn) {
     this.password = inputEl.value;
   }
 
+  checkLogin(e) {
+    if (e.key == "Enter") this.loginConfirm();
+  }
+
   setVisibility() {
-    this.pswVisibility = !this.pswVisibility
+    this.pswVisibility = !this.pswVisibility;
+  }
+
+  getElement(query) {
+    return this.renderRoot.querySelector(query) ?? null;
   }
 
   loginConfirm() {
+    if (
+      this.getElement("#username").value === "" &&
```

Comment:
non è necessario andare a recuperare l'elemento per poi leggerne il valore, il contenuto viene sincronizzato costantemente in this.username tramite onUsernameInput